DigitalLearningSolutions.Data/Models/Email/Email.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace DigitalLearningSolutions.Data.Models.Email
+{
+    using MimeKit;
+
+    public class Email
+    {
+        public string[] To, Cc, Bcc;
+        public string Subject;
+        public BodyBuilder Body;
+
+        public Email()
```

Comment:
I feel like the subject, body and recipient should be required fields if you want to create an Email, so maybe we should only have two constructors:
1. string subject, BodyBuilder body, string to, string[]? to, string[]? bcc
2. string[] subjects, BodyBuilder body, string to, string[]? to, string[]? bcc